config/allowed_licenses.json

```diff
@@ -0,0 +1,2832 @@
+{
+  "name": "allowList",
+  "version": 1,
+  "product": "ecu-test-plugin",
+  "component": {
+    "group": "de.tracetronic.jenkins.plugins",
+    "name": "ecutest",
+    "version": "2.34-SNAPSHOT",
+    "licenses": [
+      {
+        "license": {
+          "id": "BSD-3-Clause"
+        }
+      }
+    ]
+  },
+  "components": [
+    {
+      "group": "com.jacob",
+      "name": "jacob",
+      "version": "1.20",
+      "licenses": [
+        {
+          "license": {
+            "id": "GNU Library or Lesser General Public License (LGPL)"
+          }
+        }
+      ]
+    },
+    {
+      "group": "com.jacob",
+      "name": "jacob",
+      "version": "1.20",
+      "licenses": [
+      ]
+    },
```

Comment:
Entry exists twice

Reply:
That's right. Otherwise the comparison to the length of license elements cannot be made